client/src/routes/Session/hooks/useLeaveSession.ts

```diff
@@ -7,21 +7,24 @@ import {NativeStackNavigationProp} from '@react-navigation/native-stack';
 import {DailyContext} from '../../../lib/daily/DailyProvider';
 import useSessionState from '../state/state';
 import {TabNavigatorProps} from '../../../lib/navigation/constants/routes';
+import useSessions from '../../Sessions/hooks/useSessions';
 
 type ScreenNavigationProps = NativeStackNavigationProp<TabNavigatorProps>;
 
 const useLeaveSession = () => {
   const {t} = useTranslation('Component.ConfirmExitSession');
   const {leaveMeeting} = useContext(DailyContext);
   const {navigate} = useNavigation<ScreenNavigationProps>();
+  const {fetchSessions} = useSessions();
 
   const resetSession = useSessionState(state => state.reset);
 
   const leaveSession = useCallback(async () => {
     await leaveMeeting();
     resetSession();
+    fetchSessions();
```

Comment:
@gewfy what do you think? I added this cause I noticed that the session that I was in still remained on the sessions list, and I guess it doesn't hurt to refresh it

Reply:
Agree! Makes sense :)